compiler/rustc_passes/src/weak_lang_items.rs

```diff
@@ -64,7 +64,10 @@ fn verify<'tcx>(tcx: TyCtxt<'tcx>, items: &lang_items::LanguageItems) {
             if item == LangItem::PanicImpl {
                 tcx.sess.err("`#[panic_handler]` function required, but not found");
             } else if item == LangItem::Oom {
-                tcx.sess.err("`#[alloc_error_handler]` function required, but not found");
+                if !tcx.features().default_alloc_error_handler {
+                    tcx.sess.err("`#[alloc_error_handler]` function required, but not found.");
+                    tcx.sess.err("Use `#![feature(default_alloc_error_handler)]` for a default error handler.");
```

Comment:
Could this be a `note` or suggestion instead?

Reply:
changed to `tcx.sess.note_without_error`